### PR TITLE
Call 'minetest.global_exists' to check for 'intllib' mod

### PR DESCRIPTION
**Pull request by AntumDeluge**
Fixes 'Undeclared global variable "intllib"'.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -1,7 +1,9 @@
 -- boilerplate to support localized strings if intllib mod is installed
 
+local have_intl = minetest.global_exists("intllib")
+
 local S
-if intllib then
+if have_intl then
```
Comment (HybridDog) [inline init.lua]: if minetest.global_exists("intllib") then

**Comment by AntumDeluge** [comment]
Yes, I suppose that is smarter since it is only called once.

File: init.lua (hunk 1 above)
Comment (AntumDeluge) [inline init.lua]: Yes, I suppose that is smarter since it is only called once.

**Comment by AntumDeluge** [comment]
I'm fairly new to pull requests on GitHub. Is creating a new commit, like I did, okay. Or is it better to reset & overwrite branch so that unaccepted commits are not not in history? Or will that mess up the pull request?

**Comment by HybridDog** [comment]
If you adjust your changes, you can run `git add -A && git commit --amend`; with the amend flag it doesn't add a new commit but changes the last one.
If you make different changes and for any reason don't want to open another PR, you can add another commit.
If you want to reorder commits and put some together, run git rebase -i and squash, move, rename, etc.

**Comment by AntumDeluge** [comment]
Okay, I'm understanding now. If I amend & force overwrite the remote branch it won't affect the PR because it is in relation to the branch, not the commit itself.

**Comment by HybridDog** [comment]
Yes, for pull requests you likely often need to force-push.
However, if the PR is closed, you can't reopen it after force-pushing (my experience ;¬).

**Comment by AntumDeluge** [comment]
Thanks again for the advice @HybridDog.

**Comment by bdjnk** [comment]
Thanks @AntumDeluge. Much appreciated.

**Pull request merged**